src/operations/insert.ts

```diff
@@ -100,7 +100,7 @@ export class InsertManyOperation extends AbstractOperation<InsertManyResult> {
     super(options);
 
     if (!Array.isArray(docs)) {
-      throw new MongoDriverError('docs parameter must be an array of documents');
+      throw new MongoInvalidArgumentError('Parameter "docs" must be an array of documents');
```

Comment:
Okay this is better than the `The xx option must be` in the other comment 🚀

Reply:
I'm realizing I made some of them be in the form `Parameter "xx" must ..."` and some of them in the form `Argument "xx" must ...`

I think I'm going to change all of them to be in the second form to match better with the naming of `MongoInvalidArgumentError`